CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ option(USE_CUDA OFF "Set USE_CUDA if you want to run tests for NVCC")
 option(BUILD_DOCS OFF "Set to build documentation")
 set(LOG_LEVEL "INFO" CACHE STRING "Log level options include TRACE, DEBUG, INFO, WARN, ERROR, CRITICAL, and OFF")
 set(CXX_STANDARD 20 CACHE STRING "Uses C++20 or 23")
+set(DSPDLOG_FMT_EXTERNAL "ON")
```

Comment:
This should be a compile_definition not a CMAKE variable. It can be replaced with `target_compile_definitions(rome PUBLIC SPDLOG_FMT_EXTERNAL=ON)`

Reply:
That's my bad. Will fix and make a new commit.